components/automate-chef-io/Makefile

```
@@ -17,8 +17,19 @@ clean_swagger_files:
 	rm -rf ./$(SWAGGER_DIR)/*
 
 sync_swagger_files: clean_swagger_files
-	pushd ../../api/external && find . -name '*.swagger.json' | cpio -p -dumv ../../components/automate-chef-io/$(SWAGGER_DIR) && popd
-	pushd ../automate-gateway/api/ && find . -name '*.swagger.json' | cpio -p -dumv ../../automate-chef-io/$(SWAGGER_DIR) && popd
+	pushd ../../api/external && \
+		find . -name '*.swagger.json' | sed 's/^\.\///' | while read i ; do \
+			mkdir -p ../../components/automate-chef-io/$(SWAGGER_DIR)/`dirname $$i` && \
```

Safer to quote the argument in case of embedded spaces:
```suggestion
			mkdir -p "../../components/automate-chef-io/$(SWAGGER_DIR)/`dirname $$i`" && \
```